proposals/devcontainer-features-distribution.md

```diff
@@ -131,6 +131,8 @@ A feature can be referenced directly in a user's [`devcontainer.json`](/docs/spe
 
 To aid in feature authorship, or in instances where a feature should not be published externally, individual features can be referenced locally from the project's file tree.
 
-A feature can be referenced directly in a user's [`devcontainer.json`](/docs/specs/devcontainer-reference.md#devcontainerjson) by relative path _inside_ the project directory.  A local feature may not be referenced outside of the project directory (`../` is not allowed), nor is an absolute path allowed.  
+A local feature is referenced in a user's [`devcontainer.json`](/docs/specs/devcontainer-reference.md#devcontainerjson) by relative path. A local feature is always referenced relative to the project root.
```

Comment:
Other paths in the devcontainer.json (e.g., "dockerfile", "dockerComposeFile", "context) are relative to the devcontainer.json's folder. Let's do the same here for consistency.

Reply:
So that I understand the behavior:

In @samruddhikhandale 's case with the devcontainer/images repo (that has a [`.devcontainer.json` in the `workspaceFolder`](https://github.com/devcontainers/images/tree/main/src/php)) this change will result in the same path being calculated (so no change needed on her side).

For example, if she had a `.devcontainer/devcontainer.json` in the `workspaceFolder` [for this image](https://github.com/devcontainers/images/tree/main/src/php), then all local features would need to be prefixed with `../`.  

@chrmarti, how do you normally pass the information on where the devcontainer lives downstream in the reference implementation?

Reply:
@joshspicer Correct. Not sure what you mean by downstream, but the config object has a `configFilePath` pointing at the file (currently using a URI for historical reasons).